mw-webapp/src/dataAccessLogic/getDayReports.ts

```diff
@@ -0,0 +1,56 @@
+import {dayReportDTOToDayReportConverter} from
+  "src/dataAccessLogic/DTOToBusinessConverter/dayReportDTOToDayReportConverter";
+import {getCurrentProblems} from "src/dataAccessLogic/getCurrentProblems";
+import {getJobsDone} from "src/dataAccessLogic/getJobsDone";
+import {getPlansForNextPeriod} from "src/dataAccessLogic/getPlansForNextPeriod";
+import {CurrentProblem} from "src/model/businessModel/CurrentProblem";
+import {DayReport} from "src/model/businessModel/DayReport";
+import {JobDone} from "src/model/businessModel/JobDone";
+import {PlanForNextPeriod} from "src/model/businessModel/PlanForNextPeriod";
+import {DayReportService} from "src/service/DayReportService";
+
+const FIRST_INDEX = 0;
```

Comment:
useless variable

Reply:
Added Todo comment

Reply:
why ? it is easier to remove it than to create a comment, please get rid